### PR TITLE
Filter descriptions and options instructions updates to README.

### PR DESCRIPTION
**Pull request by mpatek**
See #16

### CODE REVIEW

**Comment by mpatek** [comment]
@danielbachhuber review/merge please.

**Comment by danielbachhuber** [comment]
@mpatek these should be added to `readme.txt.` Once you've done so, you can run `grunt readme` to regenerate `readme.md`

**Comment by mpatek** [comment]
@danielbachhuber OK - edits are to readme.txt now. Also including instructions on setting options via filter.

File: readme.txt [hunk 1]
```diff
@@ -38,4 +38,22 @@ If you've chosen to secure your web application with an API key (the GIF2HTML5_A
...
+define( 'MYSITE_GIF2HTML5_API_URL', 'https://my-web-app.herokuapp.com/convert' );
+define( 'MYSITE_GIF2HTML5_API_KEY', 'secret-api-key' );
+...
+add_filter( 'pre_option_gif2html5_api_url', function() { return MYSITE_GIF2HTML5_API_URL } );
+add_filter( 'pre_option_gif2html5_api_key', function() { return MYSITE_GIF2HTML5_API_KEY } );
+```
+
 Now you should be ready to use the plugin.
+
+== Filters ==
```
Comment (danielbachhuber) [inline readme.txt]: Can you make this `<h3>` ?

File: readme.txt [hunk 2]
```diff
@@ -38,4 +38,22 @@ If you've chosen to secure your web application with an API key (the GIF2HTML5_A
...
+
+The plugin defines two filters.
+
+The `gif2html5_mp4_url` can be used to modify the URL returned from the `Gif2Html5::get_mp4_url` method. Functions added to this filter take the URL as a first argument, and the attachment ID as a second optional argument.
+
+The `gif2html5_snapshot_url` can be used to modify the URL returned from the `Gif2Html5::get_snapshot_url` method. Functions added to this filter take the URL as a first argument, and the attachment ID as a second optional argument.
```
Comment (danielbachhuber) [inline readme.txt]: Can you explain why you might use the filters?

**Comment by mpatek** [comment]
@danielbachhuber review/merge please.

**Comment by danielbachhuber** [comment]
:+1:

**Pull request merged**